Britannia/events/BTWK_storyline_events.txt

```diff
@@ -301,27 +297,150 @@ BTWK_story_event.0103 = { # Arrived at Glevum High Council
 	}
 }
 
-BTWK_story_event.0113 = {
+BTWK_story_event.0118 = { # Event for Tewdric: Fills player in on his opinions on bringing Arthur home
+	type = character_event
+	title = BTWK_story_event.0118.t
+	desc = BTWK_story_event.0118.desc
+	theme = realm
+
+	option = {
+		name = BTWK_story_event.0118.a
+		custom_tooltip = BTWK_story_event.0118.tt_a
+		character:40 = {
+			trigger_event = BTWK_story_event.0117
+		}
+		ai_chance = {
+			base = 1000
+		}
+	}
+	option = {
+		name = BTWK_story_event.0118.b
+		character:40 = {
+			trigger_event = BTWK_story_event
```

Comment:
Unfinished event?
Maybe also a prestige malus so it doesn't seem like the obviously better choice for a player

Reply:
My bad, just forgot to finish the link. It does have a custom tooltip letting the player know that Agricola may get a hook on them if they decide to ask him (which the AI always will) If you still want to add a prestige malus to that choice though, do say.

Could also add an opinion modifier between Tewdric/Agricola and Arthur upon his return, in reference to them supporting an end to his banishment?

Reply:
Oh I was saying that the choice to *not* ask is clearly better, since it has no obvious costs while the other one has a chance of your vassal getting a hook on you. The prestige malus would be if you take the choice to not ask Agricola about it.

Opinion modifier is a good idea if you want to add that.